main.py

```
@@ -0,0 +1,479 @@
+@app.delete("/comments/{comment_id}")
+async def delete_comment(
+        comment_id: str
+):
+    return {"comment_id": comment_id}
+
+
+######### auth ############
+# 회원 로그인
+@app.post("/auth/token", response_model=user.LoginUserResponse, status_code=status.HTTP_201_CREATED)
+async def auth_token(
+        login_data: Annotated[LoginUserRequest, Body()]
```

Ruff (0.14.13)

Unused function argument: `login_data`

(ARG001)